p2p/libp2p/netMessenger_test.go

```diff
@@ -91,9 +90,10 @@ func createMockNetworkArgs() libp2p.ArgsNetworkMessenger {
 				Type: p2p.NilListSharder,
 			},
 		},
-		SyncTimer:            &libp2p.LocalSyncTimer{},
-		PreferredPeersHolder: &p2pmocks.PeersHolderStub{},
-		PeersRatingHandler:   &p2pmocks.PeersRatingHandlerStub{},
+		SyncTimer:             &libp2p.LocalSyncTimer{},
+		PreferredPeersHolder:  &p2pmocks.PeersHolderStub{},
+		PeersRatingHandler:    &p2pmocks.PeersRatingHandlerStub{},
+		ConnectionWatcherType: "print",
```

Comment:
can be an exported const in export_test.go that will be equal to the unexported const

Reply:
added to libp2p as constant and used everywhere